build/media_source/system/js/core.es6.js

```diff
@@ -53,6 +53,8 @@ const DefaultAllowlist = {
   select: ['name'],
   textarea: ['name'],
   option: ['value', 'selected'],
+  details: [],
```

Comment:
```suggestion
  details: ['open'],
```

add the open attribute as allowed so a detail could be rendered as open (expanded)